lib/src/numeric/precision.dart

```diff
@@ -0,0 +1,215 @@
+import 'dart:core';
+import 'dart:math';
+import 'dart:typed_data';
+
+/// Utilities for working with floating point numbers.
+class Precision {
```

Comment:
This should probably rather be defined in `FloatDataType` class and subclasses `Float32` and `Float64` respectively, like the similar helpers of `IntegerDataType`.